files/scripts/swss.sh

```diff
@@ -151,7 +151,7 @@ function clean_up_chassis_db_tables()
     debug "Chassis db clean up for ${SERVICE}$DEV. asic=$asic"
 
     # First, delete SYSTEM_NEIGH entries
-    num_neigh=`$SONIC_DB_CLI CHASSIS_APP_DB EVAL "
+    num_neigh=`sonic-db-cli CHASSIS_APP_DB EVAL "
```

Comment:
this is very difficult to understand the diffence between sonic-db-cli and $SONIC_DB_CLI, making code very difficult to maintain, like landmine.

Reply:
could add a different variable to use sonic-db-cli from host:
```
if [ "$DEV" ]; then
    NET_NS="$NAMESPACE_PREFIX$DEV" #name of the network namespace
    SONIC_DB_CLI="sonic-db-cli -n $NET_NS"
    SONIC_DB_CLI_HOST="sonic-db-cli"
else
    NET_NS=""
    SONIC_DB_CLI="sonic-db-cli"
    SONIC_DB_CLI_HOST="sonic-db-cli"
fi
```